packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableInserter.ts

```diff
@@ -64,6 +65,18 @@ export default function createTableInserter(
     }
 }
 
+const calculateWidth = (tr: HTMLElement) => {
+    let width = 0;
+    tr.childNodes.forEach(td => {
+        if (safeInstanceOf(td, 'HTMLTableCellElement')) {
+            const tdRect = normalizeRect(td.getBoundingClientRect());
+            const tdWidth = tdRect.right - tdRect.left;
+            width = width + tdWidth;
+        }
+    });
+    return width;
```

Comment:
Do we also need to fix vertical inserter if the first row is merged?

Reply:
And in an extreme case, if all rows are merged, what will happen? For example
```html
<table>
 <tr>
  <td rowspan=2>test</td>
  <td rowspan=2>test</td>
</t>
</table>
```

Reply:
This is the behavior 
![AllmergedRows](https://user-images.githubusercontent.com/87443959/181130250-a1ec3847-7078-4439-830d-6f5e105d6df1.gif)

Reply:
OK.

About the first question, I mean like this:
![image](https://user-images.githubusercontent.com/23065085/181342276-af2e7711-5810-457a-8d20-9e6104d30f5b.png)

It seems I can't insert column between the first and second column.

Reply:
I see, the vertical inserter was only triggered by the first row cells. I added a check  to verify if the hovered cells has merged cells above, if true, also trigger vertical inserter.